torch/testing/_internal/common_distributed.py

```diff
@@ -486,6 +494,9 @@ def init_multigpu_helper(world_size: int, backend: str):
     divided to subsets, each process only uses a subset.
     """
     nGPUs = torch.cuda.device_count()
+    if TEST_HPU:
+        nGPUs = torch.hpu.device_count()
+
```

Comment:
Following the comment above, this can be converted into a site for preparing the `DEVICE_COUNT` map?